src/pages/why.tsx

```diff
@@ -0,0 +1,412 @@
+                                        </Link>{' '}
+                                        - we refund if your thing goes viral by accident on HackerNews
+                                    </li>
+                                    <li>
+                                        We've got{' '}
+                                        <Link href="https://github.com/posthog/posthog" external>
+                                            a huge open source project
+                                        </Link>{' '}
+                                        (&gt;20k stars), and beyond this you can inspect all our code in our repo so you
+                                        can see exactly what we're doing with your data
+                                    </li>
+                                    <li>
+                                        <Link href="/sales">We don't do outbound sales</Link> - we'd rather spend our
+                                        time on more products
+                                    </li>
+                                    <li>
+                                        You can <Link href="/pricing">see all our pricing publicly</Link> - other people
+                                        don’t hide it because "iT’s a bEtTeR vAlue!"
+                                    </li>
+                                    <li>
+                                        We don’t sell products at a loss - we run the company default alive. This means
+                                        we’ve never had a layoff, and we don’t take huge risks that means we could
+                                        disappear overnight.
+                                    </li>
+                                    <li>We don’t screw you on contract terms like auto-renewal.</li>
+                                    <li>
+                                        You can use all our stuff monthly until you want to lock things in with us. We
+                                        don’t pressure people.
+                                    </li>
+                                    <li>
+                                        The way we write our content. <Link href="/posts">Just read it</Link> and you’ll
+                                        get what I mean. It’s more honest and not marketing for the sake of it - it
+                                        actively helps developers build successful products. That’s our mission.
+                                    </li>
+                                    <li>
+                                        PostHog has zero intention of selling our business. We want to see just how
+                                        crazy huge this gigantic software stack can get - and we think that it will
+                                        reach at least $100bn in value. We’ll be around and fighting for a long, long
+                                        time. It’s our life’s work.
```

Comment:
```suggestion
                                        PostHog has zero intention of selling our business. We want to see just how
                                        crazy huge this gigantic software stack can get – and we think that it will
                                        reach at least $100bn in value. We’ll be around and fighting for a long, long
                                        time. It’s our life’s work.
```